src/components/application_manager/src/commands/mobile/alert_request.cc

```diff
@@ -230,6 +230,9 @@ bool AlertRequest::PrepareResponseParameters(
   result_code = PrepareResultCodeForResponse(ui_alert_info, tts_alert_info);
   info = MergeInfos(
       ui_alert_info, ui_response_info_, tts_alert_info, tts_response_info_);
+  if (ui_alert_info.is_unsupported_resource && tts_alert_info.is_ok) {
```

Comment:
@pvvasilev maybe 

if (!result && tts_alert_info.is_ok && ui_alert_info.is_unsupported_resource) {
   result = true;
}

for consistency with the code around..